Templates/DefaultProject/Template/ExportScripts/export_standalone_monolithic.py

```diff
@@ -0,0 +1,321 @@
+            user_input = input(f'Continuing may cause build errors.\nStop them before continuing? (y/n). Quit(q)')
+            if user_input.lower() == 'y':
+                for p in processes_to_kill:
+                    p.terminate()
+                    p.wait()
+            elif user_input.lower() == 'q':
+                quit()
+
+
+def process_assets( project_name: str,
+                    project_path: pathlib.Path,
+                    non_mono_build_path: pathlib.Path):
+    process_command(["cmake", "-B", non_mono_build_path, "-S", project_path])
+
+    process_command(["cmake", "--build", non_mono_build_path, "--target", "AssetProcessorBatch", "AssetBundlerBatch", "--config", "profile", "--", "/m"])
+    process_command(["cmake", "--build", non_mono_build_path, "--target", f"{project_name}.Assets", "--config", "profile", "--", "/m"])
+
+def bundle_assets(project_path: pathlib.Path,
+                  selected_platform: str,
+                  seedlist_paths: List[str],
+                  non_mono_build_path: pathlib.Path):
+    asset_bundler_batch_path = non_mono_build_path / 'bin' / 'profile' / ('AssetBundlerBatch' + ('.exe' if platform.system().lower()=='windows' else ''))
+    asset_list_path = project_path / 'AssetBundling' / 'AssetLists'
+
+    game_asset_list_path = asset_list_path /  f'game_{selected_platform}.assetlist'
+    engine_asset_list_path = asset_list_path / f'engine_{selected_platform}.assetlist'
+    bundles_path = project_path / 'AssetBundling' / 'Bundles'
+
+    #generate the asset lists for the project
+    gen_game_asset_list_command = \
+        f"{asset_bundler_batch_path} assetLists --assetListFile {game_asset_list_path} --platform {selected_platform} --allowOverwrites"
+    for seed in seedlist_paths:
+        gen_game_asset_list_command += " --seedListFile " + str(seed)
+    process_command(gen_game_asset_list_command.split())
+
+    gen_engine_asset_list_command = \
+        f"{asset_bundler_batch_path} assetLists --assetListFile {engine_asset_list_path} --platform {selected_platform} --allowOverwrites --addDefaultSeedListFiles"
+    process_command(gen_engine_asset_list_command)
+
+    #generate the bundles. We will place it in the project directory for now, since the files need to be copied multiple times (one for each separate launcher distribution)
+    process_command([asset_bundler_batch_path, "bundles", "--maxSize", "2048", "--platform", selected_platform, "--allowOverwrites",
+                         "--outputBundlePath", bundles_path / f"game_{selected_platform}.pak",
+                         "--assetListFile", game_asset_list_path])
+    
+    process_command([asset_bundler_batch_path, "bundles", "--maxSize", "2048", "--platform", selected_platform, "--allowOverwrites",
+                         "--outputBundlePath", bundles_path / f"engine_{selected_platform}.pak",
+                         "--assetListFile", engine_asset_list_path])
```

Comment:
Looks like these two processes can be parallelized. Have you considered having a thread to signal when a process_command returns instead of waiting and blocking for the result?

Reply:
I've thought about doing that, but in practice I've noticed the engine bundle does not take long at all (only a second or so). The bottle neck is almost exclusively with the game bundle in all of the build tests I've done. As such I dont find it necessary yet.